### ISSUE TITLE
Application GUI opens blank window

### ISSUE THREAD

**Issue by hashamyounis9** [JoseExposito/touche#99]
### Describe the bug

I have installed touche in my kali linux rolling using flatpak, when i try and open the app it just shows a blank resize-able window and does 
![Screenshot from 2024-08-11 23-28-48](https://github.com/user-attachments/assets/0101c291-5b4d-4575-83df-aecf5f68eb87)
nothing

### Expected behaviour

When I open the app, it should show me this kind of gui:
![adwaita](https://github.com/user-attachments/assets/9a819c28-230e-4702-bed0-fcc1bec631b8)

### Actual behaviour

Application GUI not working properly, showing just blank window

### Logs

If required, include logs of the bug:

```
$ flatpak run com.github.joseexposito.touche

┌──(root㉿kali)-[~]
└─# flatpak run com.github.joseexposito.touche

(com.github.joseexposito.touche:2): Gjs-CRITICAL **: 23:56:02.404: JS ERROR: TypeError: GLib.getenv(...) is null
warnAboutWayland@webpack:///../src/main-view/index.js?:68:14
_init@webpack:///../src/main-view/index.js?:52:10
MainView@webpack:///../src/main-view/index.js?:35:1
_init@webpack:///../src/app-window.js?:50:21
AppWindow@webpack:///../src/app-window.js?:42:1
main/<@webpack:///../src/index.js?:49:22
main@webpack:///../src/index.js?:53:22
run@resource:///org/gnome/gjs/modules/script/package.js:207:19
@/app/bin/com.github.joseexposito.touche:11:17


### Your environment

 * Touché version: 2.0.12 - 2024-04-13
 * Touchégg version: Touchégg v2.0.17.
 * Operating System: 
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
VERSION_ID="2024.2"
VERSION="2024.2"
VERSION_CODENAME=kali-rolling
ID=kali
ID_LIKE=debian
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
ANSI_COLOR="1;31"

 * Desktop Environment: Gnome
 * Installation method: Flatpak
 * Are you using Wayland? No, I am using touchegg

**Comment by JoseExposito** [comment]
Hi @hashamyounis9 

Looking at the logs, it looks like the problem happens when the `XDG_SESSION_TYPE` environment variable is not set.

I guess you figured it out and closed the issue, but this'd need to be solved. If you don't mind, I'll reopen the issue so I don't forget to fix it.

**Comment by hashamyounis9** [comment]
Hey @JoseExposito 
I tried setting `XDG_SESSION_TYPE` manually in global environment but it did not work. I guess it needs to be set in the environment that package uses...

**Comment by JoseExposito** [comment]
I'll be fixed in 2.0.13, publishing on Flathub

**Comment by hashamyounis9** [comment]
Thanks @JoseExposito for resolving!
